2022-10-07 15:42:25,735 - INFO basePage.py [line:86]: 查找"None"元素，元素定位:('id', 'username')
2022-10-07 15:42:25,750 - INFO basePage.py [line:111]: 在"None"输入"43347",元素定位:('id', 'username')
2022-10-07 15:42:25,829 - INFO basePage.py [line:86]: 查找"None"元素，元素定位:('id', 'password')
2022-10-07 15:42:25,837 - INFO basePage.py [line:111]: 在"None"输入"BW@jason8854",元素定位:('id', 'password')
2022-10-07 15:42:26,904 - INFO basePage.py [line:86]: 查找"None"元素，元素定位:('id', 'submit')
2022-10-07 15:42:26,904 - INFO basePage.py [line:138]: 点击"None",元素定位:('id', 'submit')
2022-10-07 15:42:27,777 - INFO basePage.py [line:86]: 查找"None"元素，元素定位:('id', 'firstPageAction')
2022-10-07 15:42:27,800 - INFO basePage.py [line:138]: 点击"None",元素定位:('id', 'firstPageAction')
2022-10-07 15:42:28,828 - INFO basePage.py [line:182]: iframe 切换操作:
2022-10-07 15:42:29,834 - INFO basePage.py [line:188]: 切换成功
2022-10-07 15:42:32,849 - INFO basePage.py [line:86]: 查找"None"元素，元素定位:('id', 'insertDayPlan')
2022-10-07 15:42:32,867 - INFO basePage.py [line:138]: 点击"None",元素定位:('id', 'insertDayPlan')
2022-10-07 15:42:35,947 - INFO basePage.py [line:182]: iframe 切换操作:
2022-10-07 15:42:43,067 - ERROR basePage.py [line:190]: iframe 切换失败!!!
Traceback (most recent call last):
  File "D:\RiQingUITest\Common\basePage.py", line 186, in switch_iframe
    EC.frame_to_be_available_and_switch_to_it(frame_refer))
  File "E:\IT_Application\Python\python\lib\site-packages\selenium\webdriver\support\wait.py", line 81, in until
    value = method(self._driver)
  File "E:\IT_Application\Python\python\lib\site-packages\selenium\webdriver\support\expected_conditions.py", line 256, in _predicate
    driver.switch_to.frame(driver.find_element(*locator))
  File "E:\IT_Application\Python\python\lib\site-packages\selenium\webdriver\remote\switch_to.py", line 94, in frame
    self._driver.execute(Command.SWITCH_TO_FRAME, {'id': frame_reference})
  File "E:\IT_Application\Python\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 428, in execute
    self.error_handler.check_response(response)
  File "E:\IT_Application\Python\python\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 243, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=106.0.5249.62)
Stacktrace:
Backtrace:
	Ordinal0 [0x0089DF13+2219795]
	Ordinal0 [0x00832841+1779777]
	Ordinal0 [0x0074423D+803389]
	Ordinal0 [0x0072C70A+706314]
	Ordinal0 [0x00791502+1119490]
	Ordinal0 [0x0079E8FA+1173754]
	Ordinal0 [0x0078E616+1107478]
	Ordinal0 [0x00767F89+950153]
	Ordinal0 [0x00768F56+954198]
	GetHandleVerifier [0x00B92CB2+3040210]
	GetHandleVerifier [0x00B82BB4+2974420]
	GetHandleVerifier [0x00936A0A+565546]
	GetHandleVerifier [0x00935680+560544]
	Ordinal0 [0x00839A5C+1808988]
	Ordinal0 [0x0083E3A8+1827752]
	Ordinal0 [0x0083E495+1827989]
	Ordinal0 [0x008480A4+1867940]
	BaseThreadInitThunk [0x775FFA29+25]
	RtlGetAppContainerNamedObjectPath [0x777E7B5E+286]
	RtlGetAppContainerNamedObjectPath [0x777E7B2E+238]

